en/option/series/line.md

```diff
@@ -37,7 +37,7 @@ Broken line chart relates all the data points [symbol](~series-line.symbol) by b
 ## showSymbol(boolean) = true
 Whether to show symbol. It would be shown during tooltip hover.
 
-## showAllSymbol(boolean) = 'auto'
+## showAllSymbol(boolean|string) = 'auto'

```

Comment:
For the property which value can be `'auto'`.  I think we can ignore the `string` type. Because `'auto'` is a very special value. It's similar to `null/undefined` but with semantic.